market-service/src/main/java/rs/edu/raf/banka1/controller/MarketController.java

```diff
@@ -235,7 +235,7 @@ public ResponseEntity<ListingStockDto> getStockById(@PathVariable Long stockId)
         return new ResponseEntity<>(listingStockDto, HttpStatus.OK);
     }
 
-    @GetMapping(value = "/listing/stock/{stockId}/time", produces = MediaType.APPLICATION_JSON_VALUE)
+    @GetMapping(value = "/exchange/stock/{stockId}/time", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Da li ovo stoji i u endpoints file-u? Ako ne to bi trebalo da se ažurira.

Reply:
Stoji i u endpoints file-u.